### PR TITLE
catkinized pr2_jacobian_tests, as pr2_moveit_tests depends on this.

### PR DESCRIPTION
**Pull request by dornhege**
This should be sufficient to make CI not fail any more for moveit_pr2.

### CODE REVIEW

File: tests/pr2_jacobian_tests/CMakeLists.txt [hunk 1]
```diff
@@ -1,35 +1,9 @@
...
-#rosbuild_link_boost(${PROJECT_NAME} thread)
-#rosbuild_add_executable(example examples/example.cpp)
-#target_link_libraries(example ${PROJECT_NAME})
-
-rosbuild_add_executable(my_test src/main.cpp)
-rosbuild_declare_test(my_test)
-target_link_libraries(my_test gtest)
-rosbuild_add_rostest(test_pr2_jacobian_solver.launch)
+catkin_package(
+    DEPENDS kdl_parser
```
Comment (v4hn) [inline tests/pr2_jacobian_tests/CMakeLists.txt]: This is wrong. `DEPENDS` is only used for _non-ROS_ package dependencies.

Comment (dornhege) [inline tests/pr2_jacobian_tests/CMakeLists.txt]: I trusted catkinize too much. Will be fixed.

**Comment by dornhege** [comment]
Please hold back merging until CI works for this.

**Comment by v4hn** [comment]
CI passed. Merging.

**Pull request merged**